burn-core/src/nn/rnn/gru.rs

```diff
@@ -21,7 +21,7 @@ pub struct GruConfig {
     pub bias: bool,
     /// Gru initializer
     /// TODO: Make default Xavier initialization. https://github.com/burn-rs/burn/issues/371
-    #[config(default = "Initializer::Uniform(0.0, 1.0)")]
+    #[config(default = "Initializer::Uniform{min:0.0, max:1.0}")]
```

Comment:
Can we also change this to Xavier? This probably got in during your PR after the merge and that's why we missed it.